tests/LSP9Vault/LSP9Vault.behaviour.ts

```diff
@@ -221,6 +221,80 @@ export const shouldBehaveLikeLSP9 = (
           .withArgs(context.accounts.friend.address, disallowedAddress);
       });
     });
+
+    describe("when renouncing ownership of the vault", async () => {
+      before(async () => {
+        await network.provider.send("hardhat_mine", ["0xffff"]);
+      });
+
+      beforeEach(async () => {
+        context = await buildContext();
+      });
+
+      it("should fail to confirm if delay didn't expire", async () => {
+        const renounceOwnershipOnce = context.lsp9Vault
```

Comment:
You have to `await` here the transaction in order to wait for the transaction receipt.

```suggestion
        const renounceOwnershipOnce = await context.lsp9Vault
```